1.Core/UIElements/Interfaces/IClickable.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace Todoly.Core.UIElements.Interfaces
 {
-    public interface IClickable
+    public interface IClickable : IElement
```

Comment:
I think another option instead of doing this is to create an IButton interface that inherits from IElement and IClickable.

Reply:
Great idea, implemented.